v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md

```diff
@@ -29,31 +29,29 @@ Similar to [Homebrew](/how-to/get-started/local-cluster/install-from-homebrew.md
 
 After <a href="https://github.com/datacharmer/dbdeployer">installing DBdeployer</a>, install a MySQL 5.7 client:
 <pre>
-curl -LO https://dev.mysql.com/get/Downloads/MySQL-5.7/mysql-5.7.25-macos10.14-x86_64.tar.gz
-dbdeployer unpack mysql-5.7.25-macos10.14-x86_64.tar.gz
+dbdeployer downloads get-unpack --overwrite --delete-after-unpack mysql-5.7.26-macos10.14-x86_64.tar.gz
 </pre>
 
 Install TiDB using the installed MySQL 5.7 client:
 <pre>
 curl -O https://download.pingcap.org/tidb-master-darwin-amd64.tar.gz
 dbdeployer unpack tidb-master-darwin-amd64.tar.gz --unpack-version=3.0.0
-dbdeployer deploy single 3.0.0 --client-from=5.7.25
+dbdeployer deploy single 3.0.0 --client-from=5.7.26
 </pre>  
 
 </section>
   <section id="LinuxContent">
 
 After <a href="https://github.com/datacharmer/dbdeployer">installing DBdeployer</a>, install a MySQL 5.7 client:
 <pre>
-dbdeployer remote get mysql-5.7.25
-dbdeployer unpack mysql-5.7.25.tar.xz
+dbdeployer downloads get-unpack --overwrite --delete-after-unpack mysql-5.7.26-linux-glibc2.12-x86_64.tar.gz
 </pre>
 
-Install TiDB using the installed MySQL 5.7 client:
+Install TiDB 3.0 using the installed MySQL 5.7 client:
 <pre>
-wget https://download.pingcap.org/tidb-master-linux-amd64.tar.gz
-dbdeployer unpack tidb-master-linux-amd64.tar.gz --unpack-version=3.0.0
-dbdeployer deploy single 3.0.0 --client-from=5.7.25
+wget https://download.pingcap.org/tidb-v3.0-linux-amd64.tar.gz
```

Comment:
@zhouqiang-cl @kolbe The URL I've used here temporarily is all 3.0 components. Do we have one that is like the previous URL (version 3.0, but only tidb-server)?

Reply:
I found https://download.pingcap.org/tidb-release-2.1-linux-amd64.tar.gz but substituting 3.0 gives 404.

Reply:
@zhouqiang-cl @mahjonp should there be tidb-release-3.0-linux-amd64.tar.gz to match the 2.1 package?

```
$ curl -sSL https://download.pingcap.org/tidb-release-2.1-linux-amd64.tar.gz | tar tzvf -
drwxr-xr-x  0 pingcap staff       0 Jun  4 09:07 tidb-release-2.1-linux-amd64/
drwxr-xr-x  0 pingcap staff       0 Jun  4 09:07 tidb-release-2.1-linux-amd64/bin/
-rwxr-xr-x  0 pingcap staff 57154776 Jun  4 03:52 tidb-release-2.1-linux-amd64/bin/tidb-server
```

Reply:
@kolbe I have added a daily release job of the release-3.0 branch, and it's ok now.